### ISSUE TITLE
Names like `__some__value__` does not raise consecutive underscores violation

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#404]
# Bug report

## What's wrong

```python
__some__value__ = 1
__private__value = 1
```

It does not raise any violations.

## How is that should be

It should raise `ConsecutiveUnderscoreViolation`.
